tutorials/multi-deployment-tutorial.md

```diff
@@ -0,0 +1,92 @@
+# Terraform Google Cloud Migrate
+
+## Let's get started!
+
+This guide will show you how to deploy the infrastructure required for Migrate using Terraform and Cloud Foundation Toolkit.
+
+**Time to complete**: About 1 hour
+
+Click the **Start** button to move to the next step.
+
+## Cloning the repository
+
+Let's get started by cloning the git repo Migrate Cloud Foundations Toolkit repository inorder to use the provided examples.
+
+```bash
+git clone https://github.com/terraform-google-modules/terraform-google-migrate
+```
+
+Let's navigate to the cloned repo and into the examples.
+We will be using the multi_deployment example in this walkthrough.
+
+```bash
+cd terraform-google-migrate/examples/multi_deployment
+```
+
+## Creating the service account
```

Comment:
@morgante 
Since we are now using gcloud auth login for credentials, can we list the roles necessary for this multi deployment tutorial or should we write a command that will add the necessary roles?

Reply:
Yes, we should list the necessary roles (at a minimum).

Reply:
Listed the necessary roles from https://cloud.google.com/migrate/compute-engine/docs/4.8/how-to/configuring-gcp/configuring-gcp#setting_up_a_gcp_account_organization_and_project